app/assets/stylesheets/layout/_home.scss

```diff
@@ -103,6 +103,18 @@
 }
 
 
+.coming-soon {
+    width: 100%;
+    z-index: 29999;
+    position: relative;
+    height: 100%;
+    background: rgba(204,204,204,0.9) ;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Commas in function arguments should be followed by a single space<br>Declaration should not have a space before the terminating semicolon